lib/vsphere.rb

```diff
@@ -127,16 +134,17 @@ def validate options
     # Mebibytes for RAM and Kebibytes for disks
     options[:memory] = parse_size(options[:memory], 'M')
 
-    if options[:raw_sdb]
-      sdb_size, *sdb_path = options[:raw_sdb].split(/,/)
-      options[:sdb] = parse_size(sdb_size, 'K')
-      options[:sdb_path] = sdb_path[0]
-    end
-
     debug( 'INFO', "CPU: #{options[:cpu]}" ) if options[:debug]
     debug( 'INFO', "Mem: #{options[:memory]} MiB" ) if options[:debug]
-    debug( 'INFO', "sdb: #{options[:sdb]} KiB" ) if options[:sdb] and options[:debug]
-    debug( 'INFO', "sdb_path: #{options[:sdb_path]}" ) if options[:sdb_path] and options[:debug]
+
+    invalid_mounts = [ '/', '/boot', '/tmp', '/opt', '/var' ]
```

Comment:
We already create a partition for /tmp, which is why its in this list. We don't support secondary disks on default partitions.